core/src/com/group66/game/BustaMove.java

```diff
@@ -41,16 +41,8 @@ public int getGameWidth() {
 	 * @see com.badlogic.gdx.ApplicationListener#create()
 	 */
 	@Override
-	public void create () {
+	public void create() {
 		batch = new SpriteBatch();
 		this.setScreen(new MainMenuScreen(this));
 	}
-
-	/* (non-Javadoc)
-	 * @see com.badlogic.gdx.Game#render()
-	 */
-	@Override
```

Comment:
This overwrites the create() function and the render function
in the render function only the super.render() was passed, so I threw it out